lambdas/submit-answer/src/utils/check-details-count-calculator.ts

```diff
@@ -0,0 +1,44 @@
+import { Logger } from "@aws-lambda-powertools/logger";
+import { SubmitAnswerResult } from "../types/answer-result-types";
+
+const logger = new Logger({ serviceName: "CheckDetailsCountCalculator" });
+
+export class CheckDetailsCountCalculator {
+  public calculateCorrectAnswers(answerResults: SubmitAnswerResult[]): number {
```

Comment:
These methods could be one countAnswers and pass it a param of type for correct or incorrect